content/docs/architecture.md

```diff
@@ -2,41 +2,41 @@
 
 ## The Purpose of KUDO
 
-KUDO is built to help Dev/Ops teams manage day 2 operations of services on Kubernetes, including stateful services through the management of operators. Day 2 in this context refers to the need to support more than just installation. KUDO is built to support upgrades of services along with backup, recovery and the needs of observability. Building Kubernetes operators requires deep knowledge of the underlying service in addition to a deep knowledge of Kubernetes APIs. KUDO is built as an abstraction such that developers can build an operator without focusing on keeping up with the fast past of Kubernetes API changes. To this end KUDO supports the following:
+KUDO is built to help Dev/Ops teams manage day 2 operations of services on Kubernetes, including stateful services through the management of operators. Day 2 in this context refers to the need to support more than just installation. KUDO is built to support upgrades of services along with backup, recovery and the needs of observability. Building Kubernetes operators requires deep knowledge of the underlying service in addition to a deep knowledge of Kubernetes APIs. KUDO is built as an abstraction such that developers can build an operator without focusing on keeping up with the fast pace of Kubernetes API changes. To this end KUDO supports the following:
 
 * Create new KUDO operator packages
 * Package operators in tarballs
 * Search, install, uninstall operators
 * Perform backup, restore and upgrade plans
 * Invoke other arbitrary packaged operator plans
-* Creating operator repositories with index files
+* Create operator repositories with index files
 
 ## Architecture Diagram
 
 ![KUDO Architecture Diagram](/images/kudo-architecture.jpg?10x20)
 
 ## Components
 
-**kudoctl** or `kubectl-kudo` is a [kubectl plugin](https://kubernetes.io/docs/tasks/extend-kubectl/kubectl-plugins/) command-line client. The client is used to aid developers in creating KUDO operators and is used ops teams to manage operators in a Kubernetes cluster. The client is capable of:
+**kudoctl** or `kubectl-kudo` is a [kubectl plugin](https://kubernetes.io/docs/tasks/extend-kubectl/kubectl-plugins/) command-line client. The client is built to aid developers in creating KUDO operators and is used by ops teams to manage operators in a Kubernetes cluster. The client is capable of:
```

Comment:
now that I am re-reading it, should we actually call this KUDO CLI as we do in other parts of the docs? 🤔 I don't think we ever refer to it as kudoctl

Reply:
Right; I was confused about this at first too – I was wondering whether we just introduced a `kudoctl` command …

Reply:
Changed it to `kubectl-kudo` since it refers to the label on the diagram